src/components/Class/ClassCard/index.js

```diff
@@ -183,60 +198,137 @@ function ClassCard({ item, editClass, enroll, style }) {
       dispatch(classActions.dropOutClass(Id));
     });
   };
+
+  const EnrolledAndTimer = () => {
+    const [Timer, setTimer] = useState(TimeLeft(item.start_time));
+    const ONE_MINUTE = 60000; //millisecs
+    setInterval(() => {
+      setTimer(TimeLeft(item.start_time));
+    }, ONE_MINUTE);
```

Comment:
Few questions/points here:
1. Should this timer be being run every second instead of minute when the time gets under 1 minute or even when the time gets under 1 day?
2. By calling `setInterval` here, you will be repeatedly creating a new repeating timer with every render, which will happen at least as often as the time on the button is changed.  Instead, it'd be better to call it in a `setEffect` called once that cleans up after itself.  Something like:
```
useEffect(() => {
  const repeatingTimer =
      setInterval(() => setTimer(TimeLeft(item.start_time)), ONE_MINUTE);
  return () => clearInterval(repeatingTimer); // cleans up on unmount
}, []);
```
But if you want the time to do a variable time update (e.g., every 1 second from joining time), you can use `setTimeout` with a `[Timer]` dependency.
3. How urgent is this PR?  If it's not urgent, I can create a PR to address this and also incorporate the new function from #436 .  It shouldn't take long, but I don't know if you would need additional time to test it.